lib/yast2_shortcuts.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Shortcuts for yast modules
+# Maintainer: Joaquín Rivera <jeriveramoya@suse.com>
+
+package yast2_shortcuts;
+
+use strict;
+use warnings;
+use version_utils qw(is_leap is_sle is_opensuse);
+
+use Exporter 'import';
+our @EXPORT_OK = qw($is_newer_product %remote_admin %firewall_settings %firewall_details $confirm);
+
+our $is_newer_product = is_sle('>=15') || (is_opensuse && !is_leap('15.0'));
+our %remote_admin = (
+    allow_remote_admin_with_session    => 'alt-a',
+    allow_remote_admin_without_session => 'alt-l',
+    do_not_allow_remote_admin          => 'alt-n'
+);
+our %firewall_settings = (
+    open_port => $is_newer_product ? 'alt-f' : 'alt-p',
+    details => 'alt-d'
+);
+our %firewall_details = (
+    network_interfaces => 'alt-e',
+    select_all         => 'alt-a'
+);
+our $confirm = $is_newer_product ? 'alt-n' : 'alt-o';
+
+sub test_flags { }
```

Comment:
I had to add the flag function to pass Travis after checking in my local the offending line in the script:
```
/tools/detect_code_dups
Can't use string ("") as an ARRAY ref while "strict refs" in use at ./tools/detect_code_dups line 106.
```

Reply:
Ah, that's because you don't inherit from `basetest`, but could you try to import `testapi` instead, as we have in other libs. Also I still not sure if this is a good approach, with only this change it makes it more complex.

Reply:
ahm, yes I've just re-introduced testapi for being able to use $cmd{next}, I forgot to add it again after all testing of this ci issue, but it still fails. I need to add this hack to make it work anyway.

Reply:
I will take a look there with your changes, we are not in rush for this fix, so let's make it right. That might mean that we need to patch that verification script.

Reply:
yes, you're right, the script looks buggy. Thanks!